dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/ReferenceBean.java

```diff
@@ -79,13 +77,13 @@ public boolean isSingleton() {
 
     @Override
     @SuppressWarnings({"unchecked"})
-    public void afterPropertiesSet() throws Exception {

```

Comment:
IMO, this throws Exception need to keep. Check org.springframework.beans.factory.InitializingBean#afterPropertiesSet

Reply:
well